ports/rp2/boards/NULLBITS_BIT_C_PRO/modules/board.py

```diff
@@ -0,0 +1,5 @@
+from machine import Pin
+
+led_r = Pin("LEDR", Pin.OUT, value=1)
+led_g = Pin("LEDG", Pin.OUT, value=1)
+led_b = Pin("LEDB", Pin.OUT, value=1)
```

Comment:
If you use `machine.Signal(..., invert=True)` then the on/off methods will work in the correct manner.

Also, I suggest to use full colour names here, and match the pin name to the object name, eg:
```py
led_red = Signal("LED_RED", Pin.OUT, invert=True, value=0)
```
(this will require updating pins.csv as well)

This is for consistency, eg to match `ports/stm32/boards/PYBV10/pins.csv`.

Reply:
Also, I suggest adding `del Pin` at the end of this file so that `board.Pin` isn't confusingly exposed to the user.

Reply:
Good feedback! Definitely cleaner. I'll do that.